libs/design/src/atoms/form/input/input/input.component.scss

```diff
@@ -1,17 +1,23 @@
 @import '../../../../scss/daff-util';
 
 .daff-input {
-    height: inherit;
-    background: $white;
-    border: 1px solid $black;
-    padding: 15px;
-    margin: 0;
-    font-size: 1rem;
-    line-height: 1.5rem;
-    box-shadow: none;
+  height: inherit;
+  background: $white;
+  border: 1px solid daff-color($daff-gray, 300);
+  padding: 10px 15px;
+  margin: 0;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  box-shadow: none;
+  border-radius: 3px;
```

Comment:
This should probably have `width: 100%;` by default.

Reply:
@xelaint ?

Reply:
Agreed. Added in 2dc4b06